src/test/system_tests/ceph_s3_tests/test_ceph_s3_deploy.sh

```diff
@@ -17,7 +17,7 @@ DIRECTORY="s3-tests"
 CEPH_LINK="https://github.com/ceph/s3-tests.git"
 # using a fixed version (commit) of ceph tests to avoid sudden changes. 
 # we should retest and update the version once in a while
-CEPH_TESTS_VERSION=da91ad8bbf899c72199df35b69e9393c706aabee
+CEPH_TESTS_VERSION=997f78d58ac7b991e555328953fc8e72d912a303
```

Comment:
This looks like a relatively old commit. any reason to use this hash specifically?

Reply:
NVM, saw your comment in the description :)